src/Tizen.NUI/src/public/FrameBroker/FrameBrokerBase.cs

```diff
@@ -0,0 +1,281 @@
+﻿/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Threading.Tasks;
+using Tizen.Applications;
+using Tizen.Applications.Exceptions;
+
+namespace Tizen.NUI
+{
+    /// <summary>
+    /// Represents the Frame Broker.
+    /// </summary>
+    /// <since_tizen> 8 </since_tizen>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public abstract class FrameBrokerBase : IDisposable
+    {
+        private string LogTag = "Tizen.NUI";
+        private readonly SafeFrameBrokerHandle _handle;
+        private static Dictionary<int, Interop.FrameBroker.AppControlResultCallback> _resultCallbackMaps = new Dictionary<int, Interop.FrameBroker.AppControlResultCallback>();
```

Comment:
Is there any reason for being private but static?

Reply:
fixed